cli/src/test/java/com/box/l10n/mojito/cli/command/ExtractionCheckCommandTest.java

```diff
@@ -198,6 +198,107 @@ public void runSoftFailChecks() throws Exception {
     Assert.assertTrue(outputCapture.toString().contains("Checks completed"));
   }
 
+  @Test
+  public void runContextCommentCheckerWithExcludedFiles() {
+    getL10nJCommander()
+        .run(
+            "extract",
+            "-s",
+            getInputResourcesTestDir("source1").getAbsolutePath(),
+            "-o",
+            getTargetTestDir("extractions").getAbsolutePath(),
+            "-n",
+            "source1");
+
+    getL10nJCommander()
+        .run(
+            "extract",
+            "-s",
+            getInputResourcesTestDir("source2").getAbsolutePath(),
+            "-o",
+            getTargetTestDir("extractions").getAbsolutePath(),
+            "-n",
+            "source2");
+
+    getL10nJCommander()
+        .run(
+            "extract-diff",
+            "-i",
+            getTargetTestDir("extractions").getAbsolutePath(),
+            "-o",
+            getTargetTestDir("extraction-diffs").getAbsolutePath(),
+            "-c",
+            "source2",
+            "-b",
+            "source1");
+
+    getL10nJCommander()
+        .run(
+            "extraction-check",
+            "-i",
+            getTargetTestDir("extractions").getAbsolutePath(),
+            "-o",
+            getTargetTestDir("extraction-diffs").getAbsolutePath(),
+            "-c",
+            "source2",
+            "-b",
+            "source1",
+            "-cl",
+            "CONTEXT_COMMENT_CHECKER",
+            "-hf",
+            "CONTEXT_COMMENT_CHECKER",
+            "-co",
+            "contextCommentExcludeFilesPattern:.*file.js:21");
```

Comment:
We'll want to exclude based off either a file type e,g, `*.html` or a directory path `some/parent/dir/*`, can you add some tests for those scenarios and also add a test for the happy case flow where a file is excluded and the cli command passes as expected

Reply:
Updated